package.json

```diff
@@ -40,14 +40,15 @@
         "@aws-cdk/aws-iam": "1.119.0",
         "@aws-cdk/aws-logs": "1.119.0",
         "@aws-cdk/aws-secretsmanager": "1.119.0",
-        "@aws-cdk/core": "1.119.0",
+        "@aws-cdk/core": "^1.119.0",
```

Comment:
why this change?

Reply:
removed the ^ and looks like that was breaking the CI